lib/api/core/validation/types/enum.js

```diff
@@ -60,18 +62,34 @@ class EnumType extends BaseType {
    * @throws {PreconditionError}
    */
   validateFieldSpecification(typeOptions) {
-    if (!typeOptions.hasOwnProperty('values')) {
-      throw new PreconditionError('Option "values" is required');
+    if (!Object.prototype.hasOwnProperty.call(typeOptions, 'values')) {
+      errorsManager.throw('internal', 'validation', 'missing_values_option');
+
     }
 
-    if (!Array.isArray(typeOptions.values) || typeOptions.values.length === 0) {
-      throw new PreconditionError('Option "values" must be a non-empty array');
+    if (
+      !Array.isArray(typeOptions.values)
+      || typeOptions.values.length === 0
+    ) {
+      errorsManager.throw(
+        'internal',
+        'validation',
+        'values_option_cannot_be_empty'
```

Comment:
The last parenthesis should be on the last line.
There is other occurrences of this format mistake